Week3/homework/step2-3.js

```diff
@@ -30,7 +37,15 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
+  let i = 0;
+  if (num <= 0) {
+    return '';
+  }
```

Comment:
Why not have this `if` condition check for `for` and `while` loops as well? 🙂

Reply:
done ^_^